src/aat/resources/features/F-036/S-199.td.json

```diff
@@ -0,0 +1,157 @@
+{
+  "_guid_": "S-199",
+  "_extends_": "F-036_Base_Test_Data",
+  "title": "must return 200 and a StartTriggerResource object when case id and event trigger id are provided",
+  "specs": [
+    "an active profile in CCD",
+    "has a 200 code"
+  ],
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "headers": {
+      "Content-Type": "application/vnd.uk.gov.hmcts.ccd-data-store-api.start-event-trigger.v2+json;charset=UTF-8",
+      "Content-Encoding" : "gzip",
+      "Content-Length" : "[[ANY]]",
+      "Vary" : "[[ANY]]"
+    },
+    "body": {
+      "token": "[[ANY]]",
+      "_links": {
+        "self": {
+          "href": "[[ANY]]"
+        }
+      },
+      "case_details": {
+        "id": "[[ANY]]",
+        "jurisdiction": "AUTOTEST1",
```

Comment:
Can't we use the 3 newly created files for the test cases. FTA test data should ideally be separate from existing functional test cases.